public/app/plugins/datasource/prometheus/datasource.tsx

```diff
@@ -992,20 +994,23 @@ export class PrometheusDatasource
       [PromApplication.Mimir]: '/public/app/plugins/datasource/prometheus/img/mimir_logo.svg',
       [PromApplication.Prometheus]: '/public/app/plugins/datasource/prometheus/img/prometheus_logo.svg',
       [PromApplication.Thanos]: '/public/app/plugins/datasource/prometheus/img/thanos_logo.svg',
+      [PromApplication.VictoriaMetrics]: '/public/app/plugins/datasource/prometheus/img/vm_logo.svg',
```

Comment:
I'm adding this here, but the functionality that is rendering these images is being removed in https://github.com/grafana/grafana/pull/66198

Reply:
Does this require a frontend testDataSource function? Was the rendering of the images dependent on that?

Reply:
I'm not sure if I understand the question?